Sources/WalletConnectModal/UI/Common/AsyncImage.swift

```diff
@@ -12,7 +12,7 @@ struct AsyncImage<Content>: View where Content: View {
             
             var request = URLRequest(url: url)
             request.setValue(ExplorerAPI.userAgent, forHTTPHeaderField: "User-Agent")
-            request.setValue(WalletConnectModal.config.metadata.name, forHTTPHeaderField: "Referer")
+            request.setValue(WalletConnectModal.config?.metadata.name, forHTTPHeaderField: "Referer")
```

Comment:
Could you double check that it sets normal string instead of `Optional("...")`

Reply:
There is a unit test for this, verifying that both `User-Agent` and `Referer` headers are correctly populated.